src/pc/stats.rs

```diff
@@ -109,6 +111,32 @@ struct DisputesStats {
 	resolution_time: AvgBucket<u32>,
 }
 
+#[derive(Clone)]
+struct SkippedSlotBlock {
+	block_number: u32,
+	block_hash: H256,
+}
+
+impl SkippedSlotBlock {
+	pub fn new(update: &ParachainProgressUpdate) -> Self {
+		Self { block_number: update.block_number, block_hash: update.block_hash }
+	}
+
+	pub fn join_to_string(blocks: &VecDeque<Self>) -> String {
```

Comment:
This should be just a function, doesn't need to be part of `SkippedSlotBlock` impl.